README.md

```diff
@@ -107,6 +107,9 @@ StackStorm webhook handler.
 * ``list_pulls`` - List all pull requests for a particular repo.
 * ``review_pull`` - Create a review for the provided pull request.
 * ``merge_pull`` - Merge the provided pull request.
```

Comment:
probably good to list the actions in alphabetical order?

Reply:
:)

point taken